.eslintrc.js

```diff
@@ -24,6 +25,24 @@ module.exports = {
       env: {
         jest: true
       }
-    }
+    },
+    {
+      files: [
+        '**/*.tsx'
+      ],
+      plugins: [
+        '@typescript-eslint',
+        'jsx-a11y'
+      ],
+      extends: ['plugin:jsx-a11y/recommended'],
+      parser: '@typescript-eslint/parser',
+      rules: {
+        'camelcase': 'off',
+        'no-unused-vars': 'off',
+        'no-undef': 'off',
+        'no-use-before-define': 'off',
+        '@typescript-eslint/no-unused-vars': ['error'],
+      }
```

Comment:
I've disabled the `camelcase`, `no-undef` rule in typescript files until there is capacity for them to be addressed.

`no-unused-vars` on typescript files will throw false positives so we will disable that, and instead use `'@typescript-eslint/no-unused-vars'`